python/spacetime/managers/version_graph.py

```diff
@@ -154,4 +168,75 @@ def maintain(self, state_to_ref, merger_function):
         # First divergent.
         self.maintain_nodes(state_to_ref, merger_function, False)
         # The master line.
-        self.maintain_nodes(state_to_ref, merger_function, True)
\ No newline at end of file
+        self.maintain_nodes(state_to_ref, merger_function, True)
+
+
+class VersionGraphProcess(Process):
+
+    def __init__(self):
+        self.graph = Graph()
+        super().__init__()
+        self.manager = Manager()
+        self.result_manager = Manager()
+        self.event_manager = Manager()
+        self.read_write_queue = self.manager.Queue()
+        self.request_list = []
+
+        def create_flask_app(self):
```

Comment:
Make this function a method of VersionGraphProcess (instead of a curried function) and called `self.app = self.create_flask_app()`